phpBB/adm/style/admin.css

```diff
@@ -1670,6 +1670,112 @@ fieldset.submit-buttons legend {
 	}
 }
 
+/* Mentions and mention dropdown
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+.mention-container { /* mention-container */
+	text-align: left;
+	background-color: #ffffff;
+	border-radius: 2px;
+	box-shadow:
+		0 3px 1px -2px rgba(0, 0, 0, 0.2),
+		0 2px 2px 0 rgba(0, 0, 0, 0.14),
+		0 1px 5px 0 rgba(0, 0, 0, 0.12);
+	position: absolute;
+	z-index: 999;
+	overflow: auto; /* placed here for list to scroll with arrow key press */
+	max-height: 200px;
+	transition: all 0.2s ease;
+}
+
+.rtl .mention-container { /* mention-container */
+	text-align: right;
+}
+
+.mention-container ul { /* mention-list */
```

Comment:
this needs to be its own class of mention-list rather than targeting an HTML selector to prevent proper inheritance. It should be failing styleint. This will probably require modifying the JS to add the class dynamically. something like `$('.mention-container').children('ul').toggleClass('mention-list')`

Reply:
I don't currently see an option in zurb tribute to suppor this and I also don't see an option on how this could be overridden.

Reply:
It's not in zurb but you should be able to add the functionality via js like I mentioned above on call parse and add it essentially

Reply:
Done